src/main/java/com/undertheriver/sgsg/common/domain/BaseEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.undertheriver.sgsg.common.domain;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@Data
+@MappedSuperclass
+@EntityListeners({AuditingEntityListener.class})
+public class BaseEntity {
+
+	@CreatedDate
+	private LocalDate createdAt;
+
+
+	@LastModifiedDate
+	private LocalDate updatedAt;
+
+	private boolean deleted;
```

Comment:
```suggestion
	private Boolean deleted;
```
성찬님 프리미티브 타입은 nullable이 안되서.. 레퍼런스타입으로 수정부탁드려요!!